pkg/kubelet/kuberuntime/kuberuntime_sandbox_windows_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
 )
 
-func TestGetPodSandboxWindowsConfig(t *testing.t) {
+func TestGetPodSandboxWindowsConfig(t *testing.T) {
```

Comment:
Hi @endocrimes, is this the right way to write a test? Just wanted to make sure before I add more.

Reply:
That looks correct to me.

If you want to test multiple configurations you can do something similar to https://github.com/kubernetes/kubernetes/blob/17556d4d63b1761eb29caabc5b804867d49e6e5a/pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go#L150-L224 (set up an array of inputs and expected outputs and loop over the test cases) to increase code reuse.